client/app/house/schema/updateAdmin.ts

```diff
@@ -0,0 +1,9 @@
+import * as z from 'zod'
+
+const updateAdminSchema = z.object({
+  userId: z.number(),
+  houseId: z.number(),
+  isAdmin: z.boolean(),
+})
```

Comment:
You actually don't use schema validation. In that case, I think it's better to define a type in the type file!

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/93/commits/dc8278b8df03efdd66c8b4f5a634147c52828987